### PR TITLE
Feature/dropdown & confirm

### PR DESCRIPTION
**Pull request by svtslv17**
- added dropdown profile menu
- added simple confirm
- change header styles
image
image
image

### CODE REVIEW

File: src/store/reducers/isAuthorized.js [hunk 1]
```diff
@@ -12,8 +12,6 @@ export default function isAuthorized(state = initialState, action) {
       return { ...state, isAuthorized: true };
     case USER_LOGIN.FAILURE:
       return { ...state, error: action.payload };
-    case USER_LOGOUT.SUCCESS:
```
Comment (nsukhotskaya) [inline src/store/reducers/isAuthorized.js:15]: Are you sure?

File: src/components/header/headerNav/HeaderNav.jsx [hunk 2]
```diff
@@ -1,23 +1,93 @@
...
+            ) : (
+              <ExpandMoreIcon color="action" />
+            )}
+          </>
+        )}
       </Box>
+      <Collapse in={isDropDownShown}>
+        <Box className="dropDownContainer">
+          <Paper>
+            <MenuList>
```
Comment (nsukhotskaya) [inline src/components/header/headerNav/HeaderNav.jsx:74]: I thinl will be better use `Menu` component

File: src/components/confirm/Confirm.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@mui/material';
+
```
Comment (nsukhotskaya) [inline src/components/confirm/Confirm.jsx:10]:
```suggestion
import { history } from '../../../store/store'
```

File: src/components/confirm/Confirm.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  acceptButtonLabel,
+}) => {
+  const handleClickNo = () => {
+    callback(false);
+  };
+
+  const handleClickYes = () => {
+    callback(true);
+  };
+
```
Comment (nsukhotskaya) [inline src/components/confirm/Confirm.jsx:25]:
```suggestion
  const handleProfileItemClick = () => {
    history.push('/profile');
  }
```

File: src/components/header/headerNav/HeaderNav.jsx [hunk 5]
```diff
@@ -1,23 +1,93 @@
...
+              <Typography variant="h6" className="dropDownTitle">
+                {userInfo.userName}
+              </Typography>
+              <Link to="/profile" className="dropDownLink">
+                <MenuItem>
+                  {getFieldLabel('header.dropdown.label.myprofile')}
+                </MenuItem>
+              </Link>
```
Comment (nsukhotskaya) [inline src/components/header/headerNav/HeaderNav.jsx:82]:
```suggestion
                <MenuItem onClick={handleProfileItemClick}>
                  {getFieldLabel("header.dropdown.label.myprofile")}
              </MenuItem>
```

File: src/components/header/headerNav/HeaderNav.jsx [hunk 6]
```diff
@@ -1,23 +1,93 @@
...
+          acceptButtonLabel={getFieldLabel('common.yes')}
+          callback={confirmLogOut}
+        />
+      )}
+      <Box
+        className="headerNav"
+        mr={smallScreen ? '10px' : '60px'}
+        onClick={handleShowButton}
+      >
+        <Avatar alt="Ivan Ivanov" className="headerNavAvatar" />
```
Comment (nsukhotskaya) [inline src/components/header/headerNav/HeaderNav.jsx:60]:
```suggestion
        <Avatar alt={userInfo.userName} className="headerNavAvatar" />
```

**Comment by nsukhotskaya** [comment]
What do you think about it?

image

File: src/store/reducers/isAuthorized.js (hunk 1 above)
Comment (svtslv17) [reply src/store/reducers/isAuthorized.js:15]: When we use "go" after clearing the localhost, the value isAuthorized is automatically set to default without calling an action

File: src/components/header/headerNav/HeaderNav.jsx [hunk 7]
```diff
@@ -1,23 +1,93 @@
...
+import { useMediaDown } from '../../utils';
+import { Confirm } from '../../confirm';
 
 function HeaderNav() {
+  const [openConfirm, setOpenConfirm] = useState(false);
+  const smallScreen = useMediaDown('sm');
   const dispatch = useDispatch();
+  const userInfo = useSelector((state) => state.userInfo.userInfo);
+  const [isDropDownShown, setIsDropDownShown] = React.useState(false);
+
```
Comment (nsukhotskaya) [inline src/components/header/headerNav/HeaderNav.jsx:28]:
```suggestion
import { history } from '../../../store/store'
```

File: src/components/header/headerNav/HeaderNav.jsx [hunk 8]
```diff
@@ -1,23 +1,93 @@
...
+  const handleShowButton = () => {
+    setIsDropDownShown(!isDropDownShown);
+  };
+
```
Comment (nsukhotskaya) [inline src/components/header/headerNav/HeaderNav.jsx:32]:
```suggestion
  const handleProfileItemClick = () => {
    history.push('/profile');
    // and please add handleMenuClose();
  }
```

**Review by nsukhotskaya — APPROVED** [review]

**Review by oleksiuk-ol — APPROVED** [review]

**Pull request merged**